src/store/modules/notification.js

```diff
@@ -89,7 +90,37 @@ export default ({
       }
     }
   },
-  actions: {},
+  actions: {
+    loadUserNotificationsSettings ({ state, dispatch, getters, rootState }) {
```

Comment:
remove the user word to loadNotificationsSettings ({ state, dispatch, getters, rootState }) {